.github/workflows/pr-checks.yaml

```diff
@@ -90,44 +85,55 @@ jobs:
           python-version: "3.7"
           architecture: "x64"
 
-      - name: Set up CLI
+      - name: Setup-cli
+        uses: ./.github/actions/setup-cli
+
+      - name: Acceptance tests
+        timeout-minutes: 60
         run: |
-          curl -Lo operator-sdk https://github.com/operator-framework/operator-sdk/releases/download/v${SDK_VERSION}/operator-sdk_linux_amd64
-          chmod +x operator-sdk
-          mv -v operator-sdk $GITHUB_WORKSPACE/bin/
+          eval $(minikube docker-env)
+          make OPERATOR_REPO_REF=$(minikube ip):5000/sbo SKIP_REGISTRY_LOGIN=true release-operator -o registry-login test-acceptance-with-bundle
 
-          curl -Lo opm https://github.com/operator-framework/operator-registry/releases/download/v${OPM_VERSION}/linux-amd64-opm
-          chmod +x opm
-          mv -v opm $GITHUB_WORKSPACE/bin/
 
-          curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v${K8S_VERSION}/bin/linux/amd64/kubectl
-          chmod +x kubectl
-          mv -v kubectl $GITHUB_WORKSPACE/bin/
+  acceptance_non_olm:
+    name: Non OLM Acceptance Tests with Kubernetes
+    runs-on: ubuntu-20.04
 
-          curl -Lo minikube https://storage.googleapis.com/minikube/releases/v${MINIKUBE_VERSION}/minikube-linux-amd64
-          chmod +x minikube
-          mv -v minikube $GITHUB_WORKSPACE/bin/
+    env:
+      EXTRA_BEHAVE_ARGS: "--tags=~@knative --tags=~@openshift --tags=non-olm --tags=~@olm"
```

Comment:
we do not need to introduce a new `non-olm` tag, all other tags are sufficient: we want to run scenarios that do not have neither of `@knative`, `@openshift` and `@olm` tags.

Reply:
Tried it without the nonolm tag and somehow the features and scenarios (all) were being skipped

Reply:
Tried it with ` --tags=~@knative --tags=~@openshift --tags=~@olm --tags=annotations` as well :/